server.js

```diff
@@ -170,6 +170,7 @@ app.use(
         scriptSrc: [
           "'self'",
           "'unsafe-inline'",
+          'cdnjs.cloudflare.com',
```

Comment:
Wonder if it would be better to include this manually using npm and webpack?

Reply:
Personally I don't think so. But, that is just an opinion and happy for whatever consensus is arrived at.